packages/edit-site/src/components/style-book/index.js

```diff
@@ -89,35 +92,23 @@ const scrollToSection = ( anchorId, iframe ) => {
 };
 
 /**
- * Parses a Block Editor navigation path to extract the block name and
- * build a style book navigation path. The object can be extended to include a category,
- * representing a style book tab/section.
+ * Parses a Block Editor navigation path to build a style book navigation path.
+ * The object can be extended to include a category, representing a style book tab/section.
  *
  * @param {string} path An internal Block Editor navigation path.
  * @return {null|{block: string}} An object containing the example to navigate to.
  */
 const getStyleBookNavigationFromPath = ( path ) => {
 	if ( path && typeof path === 'string' ) {
-		if ( path === '/' ) {
+		if (
+			path === '/' ||
+			path.startsWith( '/typography' ) ||
+			path.startsWith( '/blocks' )
```

Comment:
Should `/colors` also be here? Without, there's no scroll to top

https://github.com/user-attachments/assets/80c5a27d-6c03-4765-a995-8d527a95d606

I'll tentatively add it

Reply:
Done in [ad39bce](https://github.com/WordPress/gutenberg/pull/68071/commits/ad39bce976f0648e2b0fd047f2fe6b2d0c7cdec6)